connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/ClientE2ETestSuite.scala

```diff
@@ -938,11 +938,11 @@ class ClientE2ETestSuite extends RemoteSparkSession with SQLHelper with PrivateM
     }
     assert(e3.getMessage.contains("AMBIGUOUS_COLUMN_REFERENCE"))
 
-    val e4 = intercept[AnalysisException] {
-      // df1("i") is ambiguous as df1 appears in both join sides (df1_filter contains df1).
-      df1.join(df1_filter, df1("i") === 1).collect()
-    }
-    assert(e4.getMessage.contains("AMBIGUOUS_COLUMN_REFERENCE"))
+    // df1.join(df1_filter, df1("i") === 1) fails in classic spark due to:
+    // org.apache.spark.sql.AnalysisException: Column i#24 are ambiguous
+    assert(
```

Comment:
shall we use `checkSameResult`?

Reply:
we can not use it here, since there is another Scala Client issue

https://github.com/apache/spark/pull/45846#discussion_r1549547958

Reply:
I am going to add it additionally in the python test

Reply:
Can we create a JIRA ticket to report this bug and leave a TODO here?

Reply:
done